### DISCUSSION TITLE
dev wallets

### DISCUSSION THREAD

**Discussion by mrheat** [nofee-ng/nofee-ng#1] [General]
I did this with scapy before but it's ridiculous you musst know all the wallet addresses, which were not easy to find... how did you do this?

**Comment by nofee-ng** [comment]
its complecated

**Comment by jd1900** [comment]
I would like to know this as well for t-rex

**Reply by nofee-ng** [reply]
t-rex is not tested right now because I don't have a Nvidia card.

**Reply by jd1900** [reply]
Any way I can help you to support it?
I've seen it connect to their servers to get the pools and addresses.

**Reply by nofee-ng** [reply]
I'll release an SSL version with debug message soon. After that, I'll contact you
